### ISSUE TITLE
Fish does not appear in backpack after catching it with a fishing pole

### ISSUE THREAD

**Issue by kasildun** [StrangeLoopGames/EcoIssues#5155]
Only after relogging it shows up in my inventory.

**Comment by MarjieAVolk** [comment]
Closing as duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/4588
